.travis.yml

```diff
@@ -153,25 +151,27 @@ before_install:
            brew unlink gcc
            brew install gcc@8
            brew link gcc@8
+           brew uninstall numpy
            find /usr/local/Cellar/ -name Python\.h
            find /usr/local/Cellar/ -name \*python\*dylib
          fi
-
-         if [ "$TRAVIS_OS_NAME" != "osx" ]; then
-           # Upgrade pip and setuptools
-           pip install --upgrade pip setuptools
-           pip install nose
-         else
-           # updating pip breaks on osx, must install man-page free nose
-           wget https://arken.nmbu.no/~plesser/nose-1.3.7noman-py3-none-any.whl
-           pip install nose-1.3.7noman-py3-none-any.whl
-         fi
+         # Upgrade pip and setuptools
+         pip install --upgrade pip setuptools
          # Installing additional packages using pip as they only have
          # outdated versions in the Travis package whitelist.
          # terminaltables is required by parse_travis_log.py to create
          # the build summary.
-         pip install cython scipy matplotlib terminaltables junitparser mpi4py
-         pip install numpy==1.16.5
+         pip install scipy junitparser mpi4py
+         if [ "$TRAVIS_OS_NAME" != "osx" ]; then
+           pip install numpy nose
+           pip install cython matplotlib terminaltables
+         else
+           # Due to the issue with numpy C-extensions, have to stick to a speciic version on OsX	 
```

Comment:
Could you add some more information, a link to an issue or a website about this problem? Just so we remember next year what this was about.

Reply:
Thanks for the review. Only after some trial and error I figured out that one specific version of numpy works on OsX. So I don't have any link for this issue. Instead I have https://numpy.org/devdocs/user/troubleshooting-importerror.html